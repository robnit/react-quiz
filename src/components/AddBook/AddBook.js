import React from 'react'

const AddBooks = () => (
  <h1>AddBooks</h1>
)

export default AddBooks