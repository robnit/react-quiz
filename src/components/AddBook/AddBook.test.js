import React from 'react'
import AddBook from 'components/AddBook/AddBook'
import { shallow } from 'enzyme' 

describe('app', () => {
  it('should match snapshots', () => {
    const wrapper = shallow(<AddBook/>)
   
    expect(wrapper).toMatchSnapshot()
  })
})
