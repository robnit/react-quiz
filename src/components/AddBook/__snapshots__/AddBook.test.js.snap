// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app should match snapshots 1`] = `
<h1>
  BOOKS
</h1>
`;
